### DISCUSSION TITLE
format of `poetry.lock`

### DISCUSSION THREAD

**Discussion by jgrecbio** [python-poetry/poetry#9403] [Q&A]
By any chance, would anyone know where I could read in the repo what is the format of `poetry.lock` ?
I know there are some info [here](https://github.com/orgs/python-poetry/discussions/6763), but I'd like to make sure.
Thx

**Comment by dimbleby** [comment]
no.  But this file is a poetry-internal detail so if you are relying on its format in any way - the chances are you don't want to do that.

**Reply by jgrecbio** [reply]
I want to have parse it to extract some data from the dependancies. Do you see a better source ?
